builds/e2e/longhaul.yaml

```diff
@@ -83,6 +83,7 @@ jobs:
       # Deploy long haul
       - template: templates/longhaul-deploy.yaml
         parameters:
+          testName: 'LongHaul'
```

Comment:
How is the `testName` different than a built-in-predefined variable `Build.DefinitionName` in this particular use case?
https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml

Reply:
We want this name as LongHaul no matter what pipeline it is run in